data/rule_data.yml

```diff
@@ -215,3 +215,6 @@ rule_data:
   disallowed_platform_patterns:
   - .*root.*
 
+  allowed_rpm_signature_keys:
+  - 199e2f91fd431d51
```

Comment:
Does the signature have a name? If so then include it in a comment.

Reply:
Done! Let me know what you think.

Reply:
Lgtm.